wordpress-mobile/WordPress-iOS
# Jetpack App: Remove the Connect Site option in the Login Epilogue

## Description

This PR removes the connect site option shown in the Login Epilogue (for the Jetpack app).

Jetpack | WordPress
--- | ---
<img src="https://user-images.githubusercontent.com/6711616/113534571-115ccb00-960c-11eb-9e57-552b7da7c703.png" width=300> | <img src="https://user-images.githubusercontent.com/6711616/113534572-13268e80-960c-11eb-8d5a-7bdebce790ca.png" width=300>

## How to test
1. If logged in, log out
2. Enter credentials and login
    - Jetpack: Notice the `Connect another site` (or `Connect a site`) is NOT displayed
    - WordPress: Notice the `Connect another site` (or `Connect a site`) is displayed
3. Tap the last row (For Jetpack, that means the last row in list of sites, and for WordPress, that means the  `Connect another site` (or `Connect a site`) row
    - Jetpack: Notice nothing happens
    - WordPress: Notice the connect site view is pushed

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Built and tested both the WordPress and Jetpack schemes

3. What automated tests I added (or what prevented me from doing so)
 Visual change, no automated tests added

PR submission checklist:
